e2e/wdio/core/tests/side-navigation.e2e-spec.ts

```diff
@@ -0,0 +1,101 @@
+import {
+    click,
+    getElementArrayLength,
+    isElementDisplayed,
+    getElementClass,
+    refreshPage,
+    waitForPresent,
+} from '../../driver/wdio';
+import { SideNavigationPo } from '../../core/pages/side-navigation.po';
+
+describe('Side-navigation test suite', () => {
+    const sideNavigationPage = new SideNavigationPo();
+    const {
+        iconsExample, objectExample, compactExample, titlesExample, condensedExample,
+        navigationExample, threeLevelsExample, pragmaticalyExample, nonSelectableExample,
+        condensedObjectExample, multipleSelectedExample, expandArrow, listItemLink,
+        pointContainsSubList, expandedListPoint, expandListExample, listItem, subListItem,
+        subList, expandList, openBtn, selectChildBtn
+    } = sideNavigationPage;
+
+    beforeAll(() => {
+        sideNavigationPage.open();
+    }, 1);
+
+    beforeEach(() => {
+        refreshPage();
+        waitForPresent(sideNavigationPage.title);
+    }, 1);
+
+    it('should check list item select', () => {
+        checkIsSelected(navigationExample);
```

Comment:
can improve. make an array of the example selectors and dot something like
array.forEach(() => {
checkIsSelected(selector)
expect....
}